### PR TITLE
[dom-mc] First ParaView supporting headless rendering with Mesa to use on mult…

### PR DESCRIPTION
**Pull request by jfavre**
i made LLVM 5.0 and Mesa 17.2.8 using the hints from the VisIt (outside of CSCS) trunk build
this enables ParaView with headless support and SWR rendering with LLVM

### CODE REVIEW

**Comment by gppezzi** [comment]
@jfavre I suggest to put on hold this PR until we merge the 18.08 PE (it should be done by this afternoon if everything goes well)

**Comment by jfavre** [comment]
updated and recompiled for 18.08

COMPLETED: Installation ended successfully
== Results of the build can be found in the log file(s) /scratch/snx1600tds/jfavre/daint/software/ParaView/5.5.2-CrayGNU-18.08-OSMesa/easybuild/easybuild-ParaView-5.5.2-20180827.085309.log

**Comment by gppezzi** [comment]
retest this please

**Comment by jfavre** [comment]
retest this please

**Comment by jfavre** [comment]
retest this please

File: easybuild/easyconfigs/l/LLVM/LLVM-5.0.0-CrayGNU-18.08.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+description = """The LLVM Core libraries provide a modern source- and target-independent
+ optimizer, along with code generation support for many popular CPUs
+ (as well as some less common ones!) These libraries are built around a well
+ specified code representation known as the LLVM intermediate representation
+ ("LLVM IR"). The LLVM Core libraries are well documented, and it is
+ particularly easy to invent your own language (or port an existing compiler)
+ to use LLVM as an optimizer and code generator."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchainopts = {'dynamic': True}
```
Comment (gppezzi) [inline easybuild/easyconfigs/l/LLVM/LLVM-5.0.0-CrayGNU-18.08.eb]: Dynamic is set True by default, so in theory having this line here is optional.

We could maybe consider to change EB and unset the `craype_link_type` if we choose static (to make sure that the user environment is not affecting the build).

Comment (jfavre) [reply easybuild/easyconfigs/l/LLVM/LLVM-5.0.0-CrayGNU-18.08.eb]: if set to True by default, then it should have worked the first time. I am still puzzled why it had failed.

**Pull request merged**